src/Text/Markdown/SlamDown/Parser/Inline.purs

```diff
@@ -372,22 +413,26 @@ parseTextBox isPlainText eta template =
   where
     parseDateTimeValue = do
       date ← parseDateValue
-      PU.skipSpaces
+      (PC.try $ void $ PS.string "T") <|> PU.skipSpaces
       time ← parseTimeValue
       pure { date, time }
 
     parseDateValue = do
       year ← natural
```

Comment:
Maybe we should require 4 digit years? It means a bit of faff as we can't rely on `natural`, but it would ensure that everything is definitely being interpreted correctly.